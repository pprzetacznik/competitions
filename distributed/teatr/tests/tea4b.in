90000000 2 80 -1 1000000 4811657015553766961
