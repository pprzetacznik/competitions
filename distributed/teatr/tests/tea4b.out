3536984095307623
